docs/classes/ERC725.md

```diff
@@ -6,27 +10,29 @@
 erc725.decodeData(data);
 ```
 
-In case you are reading the key-value store from an ERC725 smart-contract key-value store
-without `@erc725/erc725.js` you can use `decodeData` to do the decoding for you.
+If you are reading the key-value store from an ERC725 smart-contract key-value store without the `@erc725/erc725.js` library you can use the `decodeData` function to do the decoding for you.
```

Comment:
I know you didn't edit the language here but I'm confused by this line. How could you use `decodeData` function without using the @erc725/erc725.js library? 

I think maybe it should just be:
`If you are reading the key-value store from an ERC725 smart-contract you can use the `decodeData` function to do the decoding for you.`